.env.sample

```
@@ -0,0 +1,11 @@
+PORT=portnumber
+
+DATABASE_URL="mysql://username:password@127.0.0.1:3306/database_name"
+
+TYPEORM_CONNECTION=mysql
```

어떤 데이터베이스를 사용하지는지 명시하기보단 데이터베이스의 타입이 들어간다 정도로 적어주면 좋을 것 같습니다.

```suggestion
TYPEORM_CONNECTION=rdbms_type
```